generated/nifake/nifake.proto

```diff
@@ -129,6 +129,15 @@ enum NiFakeInt32AttributeValues {
   NIFAKE_INT32_BLACK = 42;
 }
 
+enum NiFakeReal64AttributeValuesMapped {
```

Comment:
It would be nice to add some nifake metadata that exercises the various scenarios this new mapping logic is supposed to support. I don't think that needs to hold up this PR though.

Reply:
Makes sense. I have created [tech debt](https://dev.azure.com/ni/DevCentral/_workitems/edit/1517663) to add nifake metadata for this.